src/components/implementation/no_interface/cFE_booter/osfiles.c

```diff
@@ -171,7 +166,7 @@ int32 OS_CloseFileByName(char *Filename)
 */
 int32 OS_mkdir(const char *path, uint32 access)
 {
-    PANIC("Unimplemented method!"); // TODO: Implement me!
+    PANIC("Read Only Filesystem!");
```

Comment:
I think it is more descriptive to describe that the file system is read only than to give a generic unimplemented method error message because "Read Only Filesystem" describes why it is not implemented and alludes to the fact that there is a whole class of functions which they should not expect to work eg. write, rm, cp